Dangerfile

```diff
@@ -10,7 +10,8 @@ end
 
 require 'json'
 
-rspec_files = Dir[File.join(ENV["CIRCLE_ARTIFACTS"], "rspec_logs_*.json")]
+containing_dir = ENV["CIRCLE_ARTIFACTS"] || "." # for local testing
```

Comment:
Is that really an issue? Did it fail your build? If the GitHub token is missing the might just fail and not even get to this point, right?